### PR TITLE
SimStore: Periodic CVs

### PR DESCRIPTION
**Pull request by dwhswenson**
This PR adds information about periodicity to SimStore CVs. It reserves the kwargs `period_min` and `period_max`.

This isn't currently used anywhere in the core of OPS, but might be used in the future to combine `PeriodicCVDefinedVolume` and `CVDefinedVolume` (probably a good option for OPS 2.0).

It will be used in the CLI for simulation setup.

### CODE REVIEW

**Comment by dwhswenson** [comment]
This is ready for review and comment. I will leave it open for at least 24 hours, merging no earlier than Sun 14 Mar 19:00 GMT (20:00 local).

File: openpathsampling/experimental/simstore/test_storable_function.py [hunk 1]
```diff
@@ -223,6 +223,24 @@ def get_expected(uuid):
...
 
+    @pytest.mark.parametrize('min_max', [(None, None), (None, 10), (0, 10)])
+    def test_check_periodic(self, min_max):
+        period_min, period_max = min_max
+        n_nones = Counter(min_max)[None]
+        expected = {2: False, 0: True, 1: 'error'}[n_nones]
+        check_period = StorableFunction._check_period
+        if expected == 'error':
+            with pytest.raises(RuntimeError):
+                check_period(period_min, period_max)
```
Comment (sroet) [inline openpathsampling/experimental/simstore/test_storable_function.py:234]: maybe make sure that the expected error is raised?
Something like:
```suggestion
            with pytest.raises(RuntimeError, match='period'):
                check_period(period_min, period_max)
```

**Review by sroet — APPROVED** [review]
LGTM in general. I would like a slightly more specific test (just to make sure the error that you expect is raised). But please don't hang this PR on that.

File: openpathsampling/experimental/simstore/test_storable_function.py (hunk 1 above)
Comment (dwhswenson) [reply openpathsampling/experimental/simstore/test_storable_function.py]: Agree, although I'm going to change this to a `ValueError`. Types `None` or numeric are allowed for `period_min` and `period_max`, but if one is `None` and the other is numeric, the values are in conflict.

Comment (sroet) [reply openpathsampling/experimental/simstore/test_storable_function.py]: > Agree, although I'm going to change this to a ValueError.

+1, I do agree that is a better error type

**Pull request merged**